slack_bolt/adapter/django/handler.py

```diff
@@ -43,9 +54,86 @@ def to_django_response(bolt_resp: BoltResponse) -> HttpResponse:
     return resp
 
 
+from django.db import connections
+
+
+def release_thread_local_connections(logger: Logger, execution_type: str):
+    connections.close_all()
```

Comment:
This method releases only the connections associated with the current thread.

see also:
* https://github.com/django/django/blob/3.1.7/django/db/__init__.py#L16
* https://github.com/django/django/blob/3.1.7/django/db/utils.py#L136-L237